### PR TITLE
Streamline idea importing

### PR DESCRIPTION
**Pull request by raniejade**
This adds a `parent` directory in our project, which should be used when importing the project into IntelliJ IDEA. This will ensure that we get proper code highlighting in the included builds. I also moved the root `.idea/*` into `parent/.idea/*`

### CODE REVIEW

File: parent/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chippmann) [inline parent/.idea/compiler.xml]: Why is this file commited?

File: parent/.idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chippmann) [inline parent/.idea/gradle.xml]: Why is this file commited?

File: parent/.idea/jarRepositories.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chippmann) [inline parent/.idea/jarRepositories.xml]: Why is this file commited?

File: parent/.idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chippmann) [inline parent/.idea/misc.xml]: Why is this file commited?

File: parent/.idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chippmann) [inline parent/.idea/vcs.xml]: Why is this file commited?

**Review by chippmann — CHANGES_REQUESTED** [review]
With this change everything works like a charm. But some comments:
- I'm no fan of the name `parent`. TBH I currently don't really have a better idea. Maybe `rootProject` or `projectRoot`?
- IMHO some documentation is missing. Because if one imports the project like you normally do, not parent is imported only `godot-kotlin`is imported.

**Comment by raniejade** [comment]
I'm not sure, I'm not a fan of `rootProject` or `projectRoot` either. Maybe `ide` or `idea`?

**Comment by raniejade** [comment]
I'll add the docs later, lets merge this to unblock @CedNaru

File: build-root/settings.gradle.kts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+includeBuild("../") {
```
Comment (raniejade) [inline build-root/settings.gradle.kts:1]: @piiertho here, this will import the root project

**Review by piiertho — APPROVED** [review]

**Review by CedNaru — APPROVED** [review]

**Review by chippmann — APPROVED** [review]
Ok for me

**Comment by raniejade** [comment]
Closing this in favor of #230

**Pull request closed without merging**